packages/bits/demo/src/components/demo/repeat/repeat-visual-test/repeat-visual-test.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="container">
+    <h3>Multiple Selection Mode with Disabled Items</h3>
+    <nui-repeat-disabled-multi-selection-example id="nui-demo-multi-disabled-items"></nui-repeat-disabled-multi-selection-example>
+    <hr />
+    
+    <h3>Multiple Selection Mode Repeating Header from Template</h3>
+    <nui-repeat-multi-selection-example id="nui-demo-multi-repeat-items"></nui-repeat-multi-selection-example>
+    <hr />
+
+    <h3>Single Selection Mode with Item Highlight</h3>
+    <nui-repeat-single-selection-mode-example id="nui-demo-single-highlight"></nui-repeat-single-selection-mode-example>
+    <hr />
+
+    <h3>Single Selection Mode with Radio Buttons</h3>
+    <nui-repeat-radio-selection-mode-example [colors]="colors" id="nui-demo-radio-selection"></nui-repeat-radio-selection-mode-example>
+    <hr />
+
+    <h3>Required Single Selection Mode</h3>
+    <nui-repeat-single-with-required-selection-mode-example id="nui-demo-single-required-selection"></nui-repeat-single-with-required-selection-mode-example>
+    <hr />
+
+    <h3>Non-Required Selection Mode with Radio Buttons</h3>
+    <nui-repeat-radio-with-non-required-selection-mode-example id="nui-demo-radio-non-required-selection"></nui-repeat-radio-with-non-required-selection-mode-example>
+    <hr />
+
+    <h3>Repeat Item Customization</h3>
```

Comment:
Maybe something like this?

```suggestion
    <h3>Item-Specific Drag/Drop Behavior</h3>
```

Reply:
Sure.